RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -247,4 +248,52 @@ bool step5(const reco::TrackBase::TrackAlgorithm& algo){
   return (algo==reco::TrackBase::tobTecStep||algo==reco::TrackBase::pixelLessStep);
 }
 
+bool goodPtResolution(const reco::TrackRef& trackref, 
+		      const std::vector<double>& DPtovPtCut,
+		      const std::vector<unsigned>& NHitCut,
+		      bool useIterTracking,
+		      bool debug) {
+  //recheck that the track is high purity!
+  if (!trackref->quality(reco::TrackBase::highPurity))
+    return false;
+    
+  const double P = trackref->p();
+  const double Pt = trackref->pt();
+  const double DPt = trackref->ptError();
+  const unsigned int NHit = 
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.